tests/qml/Properties.qml.json

```diff
@@ -5,13 +5,13 @@
       "qmlimport",
       "QtQuick",
       2.5,
-      "",
+      "QQ",
       true
     ]
   ],
   [
     "qmlelem",
-    "Rectangle",
+    "QQ.Rectangle",
```

Comment:
This is not the proper format for `qmlelem`. See [ImportsNamed.qml.json#L27-L32](https://github.com/qmlweb/qmlweb-parser/blob/master/tests/qml/ImportsNamed.qml.json#L27-L32) for how qualified `qmlelem` should be encoded. You could also test with longer paths. e.g. `Abc.Def.Rectangle`.